internal/parser/translate_httproute.go

```diff
@@ -139,6 +139,14 @@ func fromHTTPRoutes(log logrus.FieldLogger, httpRouteList []*gatewayv1alpha2.HTT
 				routes = append(routes, r)
 			}
 
+			// if there are no routes (this can happen if only unsupported match options are
+			// provided) then we'll avoid doing anything with the backendRefs.
+			if len(routes) == 0 {
```

Comment:
style mismatch: `< 1` vs `== 0`

Reply:
Actually, this is already the correct style. The `staticcheck` linter will fail if you use `< 0` due to the builtin function being unable to produce a negative value.

Example:
```
internal/parser/translate_httproute.go:42:8: SA4024: builtin function len does not return negative values (staticcheck)
				if len(rule.BackendRefs) < 0 {
```

Reply:
Gotcha, changed in 2fadbb0bbc1544c53599a8962f09c6f29bc173fb (but now I am confused why the linter picks this up for other instances but not this one)